.tours/-welcome-to-e-bikes.tour

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "👋 Welcome to E-Bikes",
+  "steps": [
+    {
+      "file": "force-app/main/default/lwc/productTileList/productTileList.js",
+      "description": "The Product Explorer lets you browse thought E-Bikes products by displaying cards that contain the product and it's price. It gets these by using an @wire to query call the getProducts Apex Class and providing the filter values that have been set. \n",
```

Comment:
Add a code tag around `getProducts` and update the sentence: it's not an Apex class, it's a method.